pl_examples/hydra_examples/conf/config.yaml

```diff
@@ -0,0 +1,22 @@
+defaults:
+  - data: mnist
+  - model: basic
+  - scheduler: step
+  - opt: sgd
+  - pl/trainer: trainer
+  - pl/logger: null
+  - pl/profiler: null
+  - pl/checkpoint: null
+  - pl/early_stopping: null
+pl:
```

Comment:
1. callbacks here probably does not belong in pl.
2. you can now create a top level user config datacalss.

Reply:
On 2, I tried to play around with doing this...is there any way to specify a subgroup without using + in the command line? For example, my command currently is : `python -m pl_examples.hydra_examples.pl_template +pl.trainer.max_epochs=5 +pl.trainer.limit_val_batches=.03 +pl.trainer.max_steps=10000 +pl.trainer.val_check_interval=.02 +pl/logger=wandb data=fashionmnist +pl.trainer.overfit_batches=.002`